CHANGES_UNRELEASED.md

```diff
@@ -2,5 +2,17 @@
 
 # Unreleased Changes
 
+## FxA Client
+
+### Breaking Changes
+
+* `migrateFromSessionToken` now returns a bool (instead of nothing), True if migration succeeded,
+False if migration failed while calling network methods.
+
+### Features
+
+* `migrateFromSessionToken` now handles offline use cases. It caches the data the consumers originally provide.
+If there's no network connectivity then the migration could be retried using the new `retryMigration` method.
```

Comment:
I don't see a `retryMigration` method, is this changelog entry from an older version of the code?